dashboard/src2/components/site/SiteMariaDBSlowQueryDialog.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+	<Dialog
+		:options="{
+			title: 'Normalized Slow Query',
+			size: '2xl'
+		}"
+		v-model="show"
+	>
+		<template #body-content>
+			<div
+				v-if="$site.doc?.version !== 'Version 14'"
+				class="mb-5 flex items-center space-x-2 rounded-lg border-blue-200 bg-blue-100 p-3 text-sm text-gray-800"
+			>
+				<i-lucide-info class="h-4 w-4" />
+				<div>Upcoming Feature : Analyze Query coming soon on Version-15</div>
+			</div>
+			<pre
+				class="rounded-lg border-2 border-gray-200 bg-gray-100 p-3 text-sm text-gray-700"
+			>
+ {{ query }} </pre
+			>
+			<div class="flex p-2 text-sm">
+				<span class="ml-auto pr-2"
+					>Duration: {{ duration.toFixed(2) }} seconds</span
+				>
+				<!-- <span class="pr-2">Rows Examined: {{ rows_examined}}</span>
+				<span class="pr-2">Rows Sent: {{rows_sent}}</span>
+				<span class="pr-2">Count: {{ count}}</span> -->
+			</div>
+		</template>
+		<template v-if="$site.doc?.version === 'Version 14'" #actions>
+			<Button
+				class="mb-4 w-full"
+				:variant="'solid'"
+				theme="gray"
+				size="sm"
+				label="Button"
+				:loading="analyze_button"
+				@click="$resources.analyzeQuery.submit()"
+			>
+				Analyze Query
+			</Button>
+			<div v-if="optimizable === true">
+				<pre
+					class="whitespace-pre-wrap rounded-lg border-2 border-gray-200 bg-gray-100 p-4 text-sm text-gray-700"
+				>
+    <span class="font-semibold">Suggested Index:</span> 
+    <span class="ml-2">Table: <span class="font-semibold">{{ this.$resources.analyzeQuery.data[0].suggested_index.split('.')[0] }}</span></span>
+    <span class="ml-2">Column: <span class="font-semibold">{{ this.$resources.analyzeQuery.data[0].suggested_index.split('.')[1] }}</span></span>
+</pre>
+
+				<Button
+					class="mb-4 w-full"
+					:variant="'solid'"
+					theme="gray"
+					size="sm"
+					label="Button"
+					@click="$resources.applySuggested.submit()"
+				>
+					Apply Suggestion
+				</Button>
+				<div v-if="sucesfullyCreatedIndexJob === true">
+					<Dialog
+						:options="{
+							title: 'Add Database Index Job',
+							message:
+								'The job to add a database index has been sucessfully created. Check the jobs tab to keep track on progress.',
+							size: 'l'
+						}"
+						v-model="show"
+					>
+					</Dialog>
```

Comment:
so whenever we do an action (a job is created in this context), we show it using a toast instead of a dialog and reroute the user to the jobs page